website/docs/docs/build/incremental-models.md

```diff
@@ -216,7 +216,7 @@ This is the behavior if `on_schema_change: ignore`, which is set by default, and
 
```

Comment:
Could you remove "and on older versions of dbt" in the sentence on line 215? This has been the default behavior for a while so is safe to remove. 

I was hoping to make it easier for you by using the built-in Add a suggestion but I can't select that line for some reason 🤷‍♀️

Reply:
Changes have been implemented as recommended.